addons/l10n_ch/report/swissqr_report.xml

```diff
@@ -15,10 +19,7 @@
                             <span class="swissqr_text content">Credit transfer</span>
                         </div>
 
-                        <img class="swissqr" t-att-src="o.invoice_partner_bank_id.build_swiss_code_url(o.amount_residual, o.currency_id.name, o.invoice_date_due, o.partner_id, 'QRR',
-                        (o.invoice_partner_bank_id.l10n_ch_isr_subscription_chf) if (o.invoice_partner_bank_id.currency_id.name == 'CHF') else o.invoice_partner_bank_id.l10n_ch_isr_subscription_eur,
-                        (o.ref) if (o.ref) else o.name)"/>
-                        <img class="ch_cross" src="/l10n_ch/static/src/img/CH-Cross_7mm.png"/>
```

Comment:
I couldn't see the cross on the report. See the attached file in the main comment.

```
>>> not o.display_swiss_qr_code() and o.invoice_display_qr_code
False
>>> o.invoice_partner_bank_id._eligible_for_qr_code('ch_qr', o.partner_id, o.currency_id)
'Street'
```